modules/ROOT/pages/dialog-components.adoc

```diff
@@ -539,6 +539,101 @@ A *textarea* is a multiline text field.
 }
 ----
 
+[[tree]]
+=== tree
+
+A *tree* is a composite component that presents an expandable category tree expandable down one level. It can also include a search and filter field.
+
+*Events:* Interacting with a *tree* component fires the `+onChange+` function in the dialog’s configuration.
```

Comment:
```suggestion
*onLeafAction:* Interacting with a *tree* component executes the `+onLeafAction+` callback set in the dialog configuration.
```
Not an event, executes a callback set using the `onLeafAction` option in the dialog spec.